src/icons/states/RelayFullEmptyState.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg/AccessibleSVG';
+import { bdlBoxBlue, bdlBoxBlue10 } from '../../styles/variables';
+
+import type { Icon } from '../flowTypes';
+
+const RelayFullEmptyState = ({ className = '', height = 140, title, width = 140 }: Icon) => (
+    <AccessibleSVG
```

Comment:
If you haven't already, I would run this through svg optimizor to see if it can become smaller